docs/services/robot-service.md

```diff
@@ -8,8 +8,11 @@ weight: 10
 tags: ["robot state", "services"]
 # SME: Cheuk
 ---
-Robot Service constitutes a minimal set of APIs that most robots (Viam Server, [Viam Python SDK](https://python.viam.dev/) and various SDKs) should support.
-Users will likely use the Robot Service as an entrypoint to interact with Viam robots and provide a way to get updates from the robot as a whole.
+
+The Robot Service refers to the minimum set of APIs that most robots running `viam-server` support.
```

Comment:
Is there a link to an interface or somethign that defines this set of APIs?

Reply:
or is that why it's still in draft and we'll do that later?

Reply:
Yeah this doc should not be reviewed. The only changes made were suggestions Sierra made that I figured I might as well implement since they were good suggestions, but I didn't mean to bring this doc into this PR at all.